src/pt/sussyscan/src/eu/kanade/tachiyomi/extension/pt/sussyscan/SussyScan.kt

```diff
@@ -9,13 +9,14 @@ class SussyScan : Madara(
     "Sussy Scan",
     "https://sussyscan.com",
     "pt-BR",
-    SimpleDateFormat("MMMMM dd, yyyy", Locale("pt", "BR")),
+    SimpleDateFormat("MMMM dd, yyyy", Locale("pt", "BR")),
 ) {
     override val client = super.client.newBuilder()
         .rateLimit(2)
         .build()
 
     override val useNewChapterEndpoint = true
 
-    override val mangaSubString = "sus"
+    // There's no manga thumbnail in manga page, keeps thumbnail from popular, latest and search
+    override val mangaDetailsSelectorThumbnail = "#thumbnail-empty"
```

Comment:
you can get the thumbnail from `og:image` in `<head>`, [like so](https://github.com/keiyoushi/extensions-source/blob/67e57a9817ce66dc9a678e7a8a81976e3e61a922/src/tr/mangaoku/src/eu/kanade/tachiyomi/extension/tr/mangaoku/MangaOku.kt#L16-L26)